ui/app/services/nomad-actions.js

```diff
@@ -0,0 +1,310 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
```

Comment:
Guess who found out that "actions" is a reserved name in Ember?

Signed, the person who just renamed this "NomadActions".